src/base/orgs/View/Projects.svelte

```diff
@@ -59,25 +60,27 @@
 </style>
 
 <div class="projects">
-  {#await seed.getProjects(profile?.id)}
+  {#await seed.getProjects(config, profile?.id)}
     <Loading center />
   {:then projects}
     {#each projects as project}
       {@const anchor = anchors[project.urn]}
       {@const pendingAnchor = pendingAnchors[project.urn]}
       {#if project.head}
         <div class="project">
-          <Widget {project} {anchor} on:click={() => onClick(project)}>
-            <span class="actions" slot="actions">
-              {#if profile?.org?.safe && account && anchor}
-                {#if pendingAnchor} <!-- Pending anchor -->
-                  <AnchorActions
-                    {account} {config} anchor={pendingAnchor} safe={profile.org.safe}
-                    on:success={() => loadAnchors()} />
+          {#await project.getCommits(project.head) then points}
```

Comment:
We shouldn't be waiting for this to load for the widget to render, ideally this is set on the widget once it is done loading in the background.

I also thought we wanted to only use the last 52 weeks of commits?

Reply:
> Another thing is that using this method forces us to fetch the project before getting its history, which shouldn't be necessary. Some options would be:

Yeah makes sense 👍 

> I also thought we wanted to only use the last 52 weeks of commits?

Yeah this escaped me, will implement it.

Reply:
Ok added the last 52 weeks of commits, and moved `getCommits` to be a class method like
`static async getCommits(urn: string, host: api.Host, parent: string, since?: string, until?: string): Promise<CommitsHistory>`